front/src/App.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div id="app">
+    <a href="https://github.com/pniedzwiedzinski/get-lucky" class="GitHub">
+      <img src="../assets/GitHub.png" alt="GitHub">
+    </a>
+    <div class="container">
+      <h1>Szczęśliwy Numerek</h1>
+      <div class="loading">
+        <span class="dot dot1"></span>
+        <span class="dot dot2"></span>
+        <span class="dot dot3"></span>
+        <h2>{{number}}</h2>
+      </div>
+      <h4>{{error}}</h4>
+      <div class="curl">
+        <h4 style="color: #DE3131;">curl </h4>
+        <h4>https://get-lucky.netlify.com/.netlify/functions/get</h4>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  name: 'app',
+  el: '#loading',
+  data() {
+    return {
+      number: null,
+      error: null,
+    }
+  },
+  mounted () {
+    this.getLuckyNumber();
+  },
+  methods: {
+    getLuckyNumber() {
+      axios.get('https://get-lucky.netlify.com/.netlify/functions/get')
+      .then(response => {
+        this.number = response.data.data;
+        const dot = document.querySelectorAll(".dot");
+        dot.forEach(item => {
```

Comment:
zrób to inaczej, niepotrzebna tutaj jest ta pętla

Reply:
A jak inaczej obsłużyć tablice od queryselectorall

Reply:
Mogę w sumie cały loading ukryć

Reply:
no